src/components/Accounts/Actions/ActionModal.js

```diff
@@ -167,14 +147,75 @@ class ActionModal extends React.Component {
     return action === 'payment';
   }
 
-  onBlurAmount = (e) => {
-    const amount = parseFloat(e.target.value || 0).toFixed(2);
-    e.target.value = amount;
+  onChangeOwner = () => {
+    const { form: { change } } = this.props;
+    change('payment-many-modal', 'method', null);
   }
 
-  onChangeOwner = () => {
-    const { dispatch } = this.props;
-    dispatch(change('payment-many-modal', 'method', null));
+  triggerCheckEndpoint = (amount, accountId) => {
+    const {
+      checkAmount,
+      okapi,
+    } = this.props;
+
+    return fetch(`${okapi.url}/accounts/${accountId}/${checkAmount}`,
```

Comment:
@annamelnyk I'm wondering did you run into some limitations with `stripes-connect` here and had to do a direct `fetch`?

Reply:
This request was implemented by @maximdidenko. As I understand direct fetch needed for continuous validation and calculations of `remaining amount` on server side while `amount` is changing in `ActionModal`